core/src/test/java/com/graphhopper/routing/util/parsers/BikeTagParserTest.java

```diff
@@ -481,12 +481,13 @@ public void testWayAcceptance() {
         way.setTag("bicycle", "yes");
         assertTrue(accessParser.getAccess(way).isWay());
 
+        // forestry is supposed to be skipped; originates in https://github.com/graphhopper/graphhopper/pull/2676
         way.clearTags();
         way.setTag("highway", "track");
         way.setTag("vehicle", "forestry");
-        assertTrue(accessParser.getAccess(way).isWay());
+        assertTrue(accessParser.getAccess(way).canSkip());
```

Comment:
To be honest, I am not sure if this really true. Just to show one example: https://www.openstreetmap.org/way/300485287

I am pretty sure that this road is accessible. I don't have good proof.

Reply:
that's just restoring the original test at https://github.com/graphhopper/graphhopper/pull/2676/files

Reply:
I know, but there was probably some reason why it was removed? https://github.com/graphhopper/graphhopper/pull/2981#pullrequestreview-2068917544